File: x264/src/main/java/com/leovp/x264/X264EncodeResult.java
package com.leovp.x264;

/**
 * Author: Michael Leo
 * Date: 21-3-18 下午4:21
 */
public class X264EncodeResult {

    public int err;
    public byte[] data;
    public long pts;
    public boolean isKey;

    public X264EncodeResult(int err, byte[] data, long pts, boolean isKey) {
        this.err = err;
        this.data = data;
        this.pts = pts;
        this.isKey = isKey;
    }
}

File: x264/src/main/java/com/leovp/x264/X264EncodeResult.kt
package com.leovp.x264

import androidx.annotation.Keep

/**
 * Author: Michael Leo
 * Date: 21-3-18 下午4:21
 */
@Keep
data class X264EncodeResult(val err: Int, val data: ByteArray, val pts: Long, val isKey: Boolean)